python/ray/dashboard/client/src/pages/dashboard/features/Workers.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { makeFeature } from "./makeFeature";
+
+const Workers = makeFeature({
+  getFeatureForNode: ({ node }) => (
+    <React.Fragment>{node.workers.length}</React.Fragment>
+  ),
+  getFeatureForWorker: ({ worker }) => (
+    <React.Fragment>{worker.cmdline[0].split("::", 2)[1]}</React.Fragment>
```

Comment:
Same about adding the comment discussing why do we need to split by `::`

Reply:
I added a comment to this effect.